_templates/plugin_layouts/sphinx_bootstrap/globaltoc.html

```diff
@@ -3,7 +3,7 @@
 <li class="dropdown globaltoc-container">
```

Comment:
See my comment on another PR about including these updates in the project_template in RunestoneComponents